docs/Makefile

```diff
@@ -1,16 +1,24 @@
-BUILDDIR    = build
-SOURCEDIR   = .
-SPHINXBUILD = sphinx-build
-SPHINXOPTS  = -a -n
+BUILDDIR     = build
+LINKCHECKDIR = $(BUILDDIR)/linkcheck
+SOURCEDIR    = .
+SPHINXBUILD  = sphinx-build
+SPHINXOPTS   = -a -n
 
-.PHONY: help clean
+.PHONY: help clean docs linkcheck
 
 help:
 	$(SPHINXBUILD) -M help $(SOURCEDIR) $(BUILDDIR) $(SPHINXOPTS)
 
 clean:
 	$(RM) -rv $(BUILDDIR)
 
+docs:
+	$(MAKE) linkcheck
+	$(MAKE) html
+
+linkcheck:
+	$(SPHINXBUILD) -b linkcheck $(SPHINXOPTS) -c $(CURDIR) $(CURDIR) build/linkcheck
```

Comment:
There is also a standalone `make linkcheck` target if you just want to iterate on link fixes during development.